xbmc/video/VideoDatabase.cpp

```diff
@@ -6668,19 +6668,22 @@ std::string CVideoDatabase::GetContentForPath(const std::string& strPath)
   {
     if (scraper->Content() == CONTENT_TVSHOWS)
     {
-      if (foundDirectly)
-        return "tvshows";
-
       // check for episodes or seasons.  Assumptions are:
       // 1. if episodes are in the path then we're in episodes.
       // 2. if no episodes are found, and content was set directly on this path, then we're in shows.
       // 3. if no episodes are found, and content was not set directly on this path, we're in seasons (assumes tvshows/seasons/episodes)
-      std::string sql = PrepareSQL("SELECT count(1) FROM episode_view WHERE strPath LIKE '%s%%' LIMIT 1", strPath.c_str());
+
+      std::string sql = "SELECT count(1) FROM episode_view ";
```

Comment:
Please use '*' instead of 1 because with '*' the sql server could choose an index for that query.
And please do SQL keywords upper-case `COUNT(*)`